src/constants/constants.js

```diff
@@ -41,89 +41,102 @@ export const usuarioPrueba = {
-  Composer de Eclipse.`,
+  numeroExpediente: '1646-9895/20',
+  numeroResolucion: 'Nro 31285/20',
+  director: 'D amato, Juan Pablo',
+  coDirector: 'Dominguez, Leonardo Daniel',
+  integrantes: [
+    'Perez, Alejandro Julian',
+    'Rubiales, Aldo Jose',
+    'Dominguez, Leonardo Daniel',
+    'D amato, Juan Pablo',
+  ],
+  resumen: `La inseguridad es un problema que afecta en mayor o menor medida a todas las ciudades 
+del mundo. Las ciudades más informatizadas hacen uso de la video-vigilancia para combatirla, montando 
+en muchos de los casos centros de monitoreo con cientos de cámaras. En su mayoría, estos centros 
+cuentan con grupos de personas para realizar la tarea de observación, sin embargo, este método no es 
+suficiente y los organismos públicos deben lidiar un reclamo social por mayor transparencia y 
+eficiencia en el accionar ante un delito. En este contexto, es que surge el presente proyecto, una 
+plataforma de administración de cámaras y sensores, para apoyar a la gestión integral de la seguridad.
+ Esta plataforma complementa técnicas de análisis automatizado de video, junto con una API para 
+registrar eventos de tipo alarmas o alertas por parte de la ciudadanía y permitir el acceso a otras 
+entidades (policía, bomberos, organizaciones vecinales) a ciertos recursos (los videos). Toda la 
+información se centraliza en un sistema georreferenciado, en una arquitectura abierta y escalable, 
+organizado en diferentes capas de información, con un sistema de organización de roles de accesos. 
+Se presenta una discusión de la estructura ideada, de los algoritmos utilizados para el seguimiento, 
+problemas propios que se suceden en este tipo de sistemas y los resultados preliminares obtenidos.`,
 };
 
 export const proyectosEnCurso = [
   {
-    nombre: 'The Phantom Menace',
-    director: 'George Lucas',
-    fechaInicio: '08-07-1999',
+    nombre:
+      'Cifrado de secuencias de vídeo e imágenes mediante código OpenCL bajo demanda',
+    director: 'Venere, Marcelo Javier',
+    fechaInicio: '20-04-2019',
     porcentaje: 98,
   },
   {
-    nombre: 'A New Hope',
-    director: 'George Lucas',
-    fechaInicio: '25-12-1977',
+    nombre:
+      'Generación de mallas de elementos finitos para arterias coronarias a partir de imágenes Ivus',
+    director: 'Jonas, Ignacio',
+    fechaInicio: '25-11-2020',
     porcentaje: 52,
   },
   {
-    nombre: 'The Force Awakens',
-    director: 'J.J. Abrahams',
-    fechaInicio: '17-12-2015',
+    nombre:
+      'Un metodo eficiente para la sustraccion de fondo en videos usando gpu',
+    director: 'Gervasoni, Luciano',
+    fechaInicio: '17-12-2020',
     porcentaje: 32,
   },
   {
-    nombre: 'Rogue One',
-    director: 'Gareth Edwards',
-    fechaInicio: '15-12-2016',
+    nombre:
+      'Plataforma de software de código abierto para aplicaciones de segmentación de imágenes médicas',
+    director: 'D amato, Juan Pablo',
```

Comment:
director: 'D\'amato, Juan Pablo',
```suggestion
    director: 'D\'amato, Juan Pablo',
```